Src/AutoFixtureUnitTest/OmitOnRecursionBehaviorTest.cs

```diff
@@ -77,5 +77,15 @@ public void TransformResultCorrectlyDecoratesInput()
             Assert.Equal(expectedBuilder, guard.Builder);
             // Teardown
         }
+
+        [Fact]
+        public void ConstructorWithRecusionDepthLowerThanOneThrows()
+        {
+            // Fixture setup
+            // Exercise system and verify outcome
+            Assert.Throws<ArgumentOutOfRangeException>(() =>
+                new OmitOnRecursionBehavior(0));

```

Comment:
Wouldn't it be more clear, that recursion depth must be lower than `1`, if we'd parameterize the test? (e.g. `0`, `-7`, `-21`)